plugin/main/src/kotlinx/benchmark/gradle/JsMultiplatformTasks.kt

```diff
@@ -46,7 +46,7 @@ private fun Project.createJsBenchmarkCompileTask(target: JsBenchmarkTarget): Kot
             )
         }
 
-        compileKotlinTask.apply {
+        compileTaskProvider.get().apply {
```

Comment:
`configure { }` instead of `get().apply { }`?

Reply:
Done